package-lock.json

```diff
@@ -2,39 +2,6 @@
   "requires": true,
   "lockfileVersion": 1,
   "dependencies": {
-    "@babel/runtime": {
```

Comment:
not sure why all of these are removed, I just installed a new dev dependency at the root of the repo